administrator/language/en-GB/com_guidedtours.ini

```diff
@@ -0,0 +1,49 @@
+; Joomla! Project
+; Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.
+; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
+; Note : All ini files need to be saved as UTF-8
+
+COM_GUIDEDTOURS_ERROR_TOUR_NOT_FOUND="Tour not found!"
+COM_GUIDEDTOURS_GUIDEDTOURS_PAGE_TITLE="Guidedtours - list of Tours"
+COM_GUIDEDTOURS_GUIDEDTOURS_TABLE_CAPTION="List of Tours"
+COM_GUIDEDTOURS_GUIDEDTOURS_LABEL_DESCRIPTION="Description"
+MOD_MENU_MANAGE_GUIDEDTOURS="Guided Tour"
+COM_GUIDEDTOURS_XML_DESCRIPTION="Component for managing Guided Tour functionality in Joomla 4!"
```

Comment:
For consistency: Component for managing Guided Tours functionality in Joomla 4 (per #24)